### DISCUSSION TITLE
TW Extension List (35+ Extensions!)

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#698] [Show and tell]
### Please check out my [Turbowarp Extension List](https://github.com/BlueDome77/Turbowarp-Extension-List) including over 30 extensions that are not listed on the Gallery!

Some sources of the extensions:

- Extensions hidden from the Gallery
- Extensions posted via issues
- Extensions uploaded through comments

Proper attribution is given to the original extension creators.

**Comment by LilyMakesThings** [comment]
I've got a pretty good extension list right here: https://github.com/TurboWarp/extensions/tree/master/extensions

**Reply by BlueDome77** [reply]
lol

**Comment by snowboyz0825** [comment]
yummy extensions to add to my pointlessly huge collection

AdmireBlocks - tons of good utilities
CN number - don't know where I saw this before, still don't know what it is
We have canvas, console, and custom reporters I think I'm not gonna bother to check if they're the same
Discord API - not smart enough to know what this does, but cool
Dotprop - dunno what this is, that formatting is disguisting though on the javascript
Seen extrautilities, as far as utility extensions go, 7/10
Global cords - seems interesting, but I feel like coding a workaround would be easier than using the extension
ISS tracker - seems broken
Images - seems broken
Inline blocks - dunno what this - holy crap that's cool
Seen javascript, too stupid to understand
Love LMS utils, that's outdated here though
Lorem ipsum - cool, but I doubt it'll ever be used
seen math and string
minecraft utils - if my laptop was powerful enough to play minecraft I'd have fun with it
misc extension - eh
Morse code - 6/10, fun, but not useful
seen everything up to splendid strings
Spledid strings - another text extension, the "remove whitespace" block is helpful though
spotify - couldn't load
Turbo Netwroking - wow that's just an old cloudlink
Turbofoolishness - nothing new, did need the millasecond block yesterday though
Vibrations - probably will use if it works, no time to check though
WebGL 2 - dunno what this is but it makes me feel powerful
Finally - nowait blocks - 3/10

**Reply by Thebloxers998** [reply]
ah yes "Netwrocking"

**Reply by Thebloxers998** [reply]
Why was my comment marked as spam

**Reply by BlueDome77** [reply]
> Why was my comment marked as spam

Github automatically marks comments as spam based off of their algorithm

**Comment by Thebloxers998** [comment]
I like the turbo networking

**Comment by Thebloxers998** [comment]
Oh ok

**Comment by milkyestway** [comment]
I believe it is a good idea to point out some extensions especially inline blocks because of its nature can crash turbowarp at least when used with other extensions(crashed pen+v7) as they were not made to handle these inputs.

**Reply by milkyestway** [reply]
also a few are outdated so I heavily recommend using the official versions of extensions(turbo networking is I believe a older version of CloudLinkV4)
